QuantitativeReporting.py

```diff
@@ -385,11 +385,14 @@ def setupSegmentationObservers(self):
       segNode.AddObserver(vtkSegmentationCore.vtkSegmentation.SegmentAdded, self.onSegmentAdded))
 
   def initializeWatchBox(self, node):
-    try:
-      dicomFileName = node.GetStorageNode().GetFileName()
```

Comment:
@pieper this line returns `None` now, but used to work before. @che85 implemented a workaround, but I wonder if  this regression could be connected to the recent re-work of the DICOM readers and if yes perhaps it should be fixed on the Slicer side?